lib/utils.js

```diff
@@ -18,7 +30,7 @@ const bNumberWithCheckDigit = (bnumber) => {
   if (remainder === 11) return `${ogBnumber}0`
   if (remainder === 10) return `${ogBnumber}x`
 
-  return `${ogBnumber}${remainder}`
+  return `${prefix}${ogBnumber}${remainder}`
```

Comment:
Comment on line 16 says 'strip prefix', but you're adding it back here?

Reply:
Yeah, we want this function to preserve the prefix (or absence of prefix).

So 
  'b1234' => 'b1234x'
  '1234' => '1234x'